tests/test_config.py

```diff
@@ -440,7 +445,7 @@ def test_match_imageversion(tmp_path: Path, op: str, version: str) -> None:
         parent = Path("mkosi.conf")
         parent.write_text(
             """\
-            [Distribution]
+            [Output]
```

Comment:
This is wrong no? Distribution should be under [Distribution]?

Reply:
Yes it should. I checked my reflog and `Distribution=fedora` was introduced in 1ad1a8b so I missed it when rebasing.